notification-portlet-webapp/src/test/groovy/org/jasig/portlet/notice/service/jpa/NotificationDTOMapperTest.groovy

```diff
@@ -0,0 +1,248 @@
+package org.jasig.portlet.notice.service.jpa;
+
+import org.dozer.DozerBeanMapper
+import org.dozer.config.BeanContainer
+import org.dozer.util.HibernateProxyResolver
+import org.hibernate.proxy.HibernateProxy
+import org.hibernate.proxy.LazyInitializer
+import org.jasig.portlet.notice.NotificationState
+import org.jasig.portlet.notice.rest.ActionDTO;
+import org.jasig.portlet.notice.rest.AddresseeDTO;
+import org.jasig.portlet.notice.rest.AttributeDTO;
+import org.jasig.portlet.notice.rest.EntryDTO
+import org.jasig.portlet.notice.rest.EventDTO
+import org.jasig.portlet.notice.rest.RecipientDTO;
+import org.jasig.portlet.notice.rest.RecipientType;
+import org.junit.Before;
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.BlockJUnit4ClassRunner
+
+import java.sql.Timestamp
+
+/**
+ * @author Josh Helmer, jhelmer.unicon.net
+ * @since 3.0
+ */
+@RunWith(BlockJUnit4ClassRunner)
+public class NotificationDTOMapperTest extends GroovyTestCase {
```

Comment:
I appreciate the unit tests on the `DTOMapper`.  I'd like to see more of this changeset covered by unit tests.